bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableTableAdminClient.java

```diff
@@ -78,4 +83,38 @@
    * @param request a {@link com.google.bigtable.admin.v2.DropRowRangeRequest} object.
    */
   void dropRowRange(DropRowRangeRequest request);
+
+  // ////////////// SNAPSHOT methods /////////////
+  /**
+   * Creates a new snapshot from a table in a specific cluster.
+   * @param request a {@link SnapshotTableRequest} object.
+   * 
+   * @return The long running {@link Operation} for the request.
+   */
+  Operation snapshotTable(SnapshotTableRequest request);
+
+  /**
+   * Get metadata information about the specified snapshot.
+   * @param request a {@link GetSnapshotRequest} object.
+   */
+  Snapshot getSnapshot(GetSnapshotRequest request);
+
+  /**
+   * Lists all snapshots associated with the specified cluster.
+   * @param request a {@link ListSnapshotsRequest} object.
+   */
+  ListSnapshotsResponse listSnapshots(ListSnapshotsRequest request);
+
+  /**
+   * Permanently deletes the specified snapshot.
+   * @param request a {@link DeleteSnapshotRequest} object.
+   */
+  void deleteSnapshot(DeleteSnapshotRequest request);
+
+  /**
+   * Creates a new table from a snapshot.
+   * @param request a {@link CreateTableFromSnapshotRequest} object.
+   */
+  Operation createTableFromSnapshot(CreateTableFromSnapshotRequest request);
```

Comment:
Add `@return The long running {@link Operation} for the request.` for consistency with `snapshotTable`